app/docs/0.11.x/network.md

```diff
@@ -44,4 +44,4 @@ Below are the recommended firewall settings:
 [proxy_listen_ssl]: /docs/{{page.kong_version}}/configuration/#proxy_listen_ssl
 [admin_api_listen]: /docs/{{page.kong_version}}/configuration/#admin_api_listen
 [admin_api_listen_ssl]: /docs/{{page.kong_version}}/configuration/#admin_api_listen_ssl
-[secure_admin_api]: /docs/{{page.kong_version}}/secure_admin_api
```

Comment:
Now that I think about it, we have the same issue with the 0.10.x version of the docs. Mind opening another PR?

Reply:
sorry, not see what youre seeing

```
$ git grep secure_admin_api
app/docs/0.11.x/network.md:* **Protect** [admin_api_listen][admin_api_listen] and [admin_api_listen_ssl][admin_api_listen_ssl], to only allow trusted sources that can access the Admin API. See also [Securing the Admin API][secure_admin_api].
app/docs/0.11.x/network.md:[secure_admin_api]: /docs/{{page.kong_version}}/secure-admin-api
```

Reply:
Hmm must have clicked a wrong link because I saw such a link at the bottom of the "Network Guide" - secure Admin API docs was written for 0.11.x iirc so yeah, sorry!